docs/docs/developer_guide/documentation.md

```diff
@@ -0,0 +1,68 @@
+# Mechaphlowers Documentation
+
+## Building documentation 
+
+### Serving documnentation
+
+From inside docs folder : 
+```{bash}
+cd ./docs
+mkdocs serve -a 0.0.0.0:8001
```

Comment:
I suggest adding here that the served documentation is then available at http://localhost:8001/ - at least that's how it works on my machine: I first tried navigating to 0.0.0.0:8001 but only got an error message. Navigating to http://localhost:8001/ worked. Does it also work with your setup?

Reply:
Done + moved  in readme